scripts/build.sh

```diff
@@ -10,126 +14,73 @@ if [ "$script_dir" == "." ]; then
 	exit 1
 fi
 
-DEV_IMAGE_NAME="devbuild"
-VERSION=${BUILD_VERSION-$DEV_IMAGE_NAME}
-
-contiv_version=${CONTIV_VERSION:-"1.0.3"}
 pull_images=${CONTIV_CI_HOST:-"false"}
 aci_gw_version=${CONTIV_ACI_GW_VERSION:-"latest"}
-ansible_image_version=${CONTIV_ANSIBLE_IMAGE_VERSION:-$contiv_version}
-auth_proxy_version=${CONTIV_API_PROXY_VERSION:-$contiv_version}
+ansible_image_version=${CONTIV_ANSIBLE_IMAGE_VERSION:-$DEFAULT_DOWNLOAD_CONTIV_VERSION}
+auth_proxy_version=${CONTIV_API_PROXY_VERSION:-$DEFAULT_DOWNLOAD_CONTIV_VERSION}
 docker_version=${CONTIV_DOCKER_VERSION:-1.12.6}
 etcd_version=${CONTIV_ETCD_VERSION:-v2.3.8}
-contiv_ansible_commit=${CONTIV_ANSIBLE_COMMIT:-4e67f54a8042debfc3d8b504046d0a1d4ea38c37}
-contiv_ansible_owner=${CONTIV_ANSIBLE_OWNER:-contiv}
 
 # the installer currently pulls the v2plugin image directly from Docker Hub, but
 # this will change to being downloaded from the Docker Store in the future.
 # because of this, the default value for this variable will become the latest
 # version that is available in the Docker Store and should be considered
 # independent of $contiv_version above.
-v2plugin_version=${CONTIV_V2PLUGIN_VERSION:-"1.0.3"}
+v2plugin_version=${CONTIV_V2PLUGIN_VERSION:-"1.1.5"}
```

Comment:
Can this be `$DEFAULT_DOWNLOAD_CONTIV_VERSION` as well by default?

Despite what the comment says, we have never had this be the latest approved version on the Docker Store... should probably delete the comment, too. lol

Reply:
no prob, testing now